shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -293,6 +312,33 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   return YES;
 }
 
+- (UniqueAotDataPtr)loadAotData:(std::string)assetsDir {
```

Comment:
If you're going to use a `std::string` (it would be much more idiomatic to pass the `NSString*` you already have available at the call site), it should be a `std::string&`.